tests/unit/testsuite/Bolt/Boltpay/controllers/Adminhtml/Sales/Order/CreateControllerTest.php

```diff
@@ -7,13 +7,56 @@
  */
 class Bolt_Boltpay_Adminhtml_Sales_Order_CreateControllerTest extends PHPUnit_Framework_TestCase
 {
+    use Bolt_Boltpay_MockingTrait;
+
     /**
      * @var
      */
     private $currentMock;
 
+    /**
+     * @var PHPUnit_Framework_MockObject_MockObject|Mage_Adminhtml_Model_Sales_Order_Create Admin Create order mock used in protected methods
+     */
+    private $adminSalesOrderCreateModelMock;
+
+    /**
+     * Ensures that the singleton for Mage_Adminhtml_Model_Sales_Order_Create is unset before running our test
+     */
+    public static function setUpBeforeClass()
+    {
+        Mage::unregister('_singleton/adminhtml/sales_order_create');
+    }
+
+    /**
+     * Registers singleton for Mage_Adminhtml_Model_Sales_Order_Create so that when Mage::getSingleton is called
+     * for this class, or mock is provided.
+     *
+     * @throws Mage_Core_Exception if there is a problem registering a singleton for Mage_Adminhtml_Model_Sales_Order_Create
+     */
     public function setUp()
     {
+        if ($this->adminSalesOrderCreateModelMock) {
```

Comment:
Maybe I'm not understanding this right, but you're assigning a value to $this->adminSalesOrderCreateModelMock when there's already a value assigned? Did you mean !$this->adminSalesOrderCreateModelMock here?

Reply:
@adentong 
Correct.  Thank you.